src/index.js

```diff
@@ -60,16 +61,14 @@ const createHoxy = () => {
     const cert = fs.readFileSync('./root-ca.crt.pem');
     opts.certAuthority = {key, cert};
   } catch (e) {
-    data.proxyStatus = 'partial';
-    data.proxyMessage = 'no HTTPS';
+    data.proxyStatus = constants.PROXY_STATUS_NO_HTTPS;
   }
 
   const hoxyServer = hoxy.createServer(opts);
   hoxyServer.on('error', (event) => {
-    data.proxyStatus = 'offline';
-    data.proxyMessage = event.code;
+    data.proxyStatus = constants.PROXY_STATUS_ERROR_GENERIC;
     if (event.code === 'EADDRINUSE') {
-      // TODO make custom window
```

Comment:
Please don't add Todos to the code. If there is something we should not forget todo create an issue for it.